staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -993,24 +993,33 @@ func (c completedConfig) New(name string, delegationTarget DelegationTarget) (*G
 func BuildHandlerChainWithStorageVersionPrecondition(apiHandler http.Handler, c *Config) http.Handler {
 	// WithStorageVersionPrecondition needs the WithRequestInfo to run first
 	handler := genericapifilters.WithStorageVersionPrecondition(apiHandler, c.StorageVersionManager, c.Serializer)
-	return DefaultBuildHandlerChainFromAuthz(handler, c)
+	return DefaultBuildHandlerChainFromAuthzToCompletion(handler, c)
 }
 
+// These handlers are split into 3, allowing us to insert additional handlers between them.
+// {DefaultBuildHandlerChain}FromStartToBeforeImpersonation
+// 							 FromImpersonationToAuthz
+// 							 FromAuthzToCompletion
```

Comment:
strange indention

Reply:
crap. My IDE renders it differently :/

Reply:
![image](https://github.com/user-attachments/assets/3cbebeb0-c01a-41b1-9a21-93270e259046)